supervision/detection/core.py

```diff
@@ -59,6 +67,11 @@ def _validate_tracker_id(tracker_id: Any, n: int) -> None:
         raise ValueError("tracker_id must be None or 1d np.ndarray with (n,) shape")
 
 
+def _validate_data_payload(data_payload: Dict[str, np.ndarray], n: int) -> None:
```

Comment:
Let's rename the function to `_validate_data`.

Reply:
Thanks! Done